src/libstd/keyword_docs.rs

```diff
@@ -1149,20 +1149,27 @@ mod where_keyword {}
 //
 /// Return a [`Future`] instead of blocking the current thread.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
-///
+/// Use `async` in front of `fn`, `closure` or a `block` to turn the marked code into a `Future`. 
```

Comment:
```suggestion
/// Use `async` in front of `fn`, `closure`, or a `block` to turn the marked code into a `Future`. 
```

Reply:
I've heard the name `oxford comma`, but never before looked up its definition. It is exactly this. Thanks - learned something new on boxing day.